commands/command.md

```diff
@@ -95,34 +122,39 @@ Command flags is @array-reply containing one or more status replies:
    4) (integer) 1
    5) (integer) 1
    6) (integer) 1
+   ...
 ```
 
-Some Redis commands have no predetermined key locations.  For those commands,
-flag `movablekeys` is added to the command flags @array-reply.  Your Redis
+Some Redis commands have no predetermined key locations. For those commands,
+flag `movablekeys` is added to the command flags @array-reply. Your Redis
 Cluster client needs to parse commands marked `movablekeys` to locate all relevant key positions.
```

Comment:
```suggestion
Some Redis commands have no predetermined key locations.
For those commands, the `movablekeys` flag is added to the command flags @array-reply,
which denotes that the `first key`, `last key`, `step` fields are insufficient to find all the keys,
and Cluster clients needs to user other measures which are described below to locate them.
```

maybe change the info in the list above to just point here rather than be an incomplete description.